File: gin/config/baidu.go
package config

/* 百度 */
type (
	TongJiType struct {
		UserName    string
		PassWord    string
		Token       string
		AccountType int
	}
)

/* 统计-商业账号 */
func TongJi() *TongJiType {
	cfg := &TongJiType{}
	cfg.UserName = "kingsoul"                      //用户名
	cfg.PassWord = "eckingsoul"                    //密码
	cfg.Token = "c67cde72015a76798c707b170fc6987e" //Token
	cfg.AccountType = 1                            //账户类型
	return cfg
}

File: java/src/main/java/webmis/config/Baidu.java
package webmis.config;

import java.util.HashMap;

/* 百度 */
public class Baidu {

  /* 统计-商业账号 */
  public static HashMap<String, Object> TongJi() {
    HashMap<String, Object> cfg = new HashMap<String, Object>();
    cfg.put("UserName", "kingsoul");
    cfg.put("PassWord", "eckingsoul");
    cfg.put("Token", "c67cde72015a76798c707b170fc6987e");
    cfg.put("AccountType", 1);
    return cfg;
  }
  
}
